src/components/PopupWithForm.js

```diff
@@ -31,10 +32,15 @@ export default class PopupWithForm extends Popup {
     super.setEveentListeners();
     this._popupForm.addEventListener("submit", (evt) => {
       evt.preventDefault();
+      this._submitButton.textContent = "Save...";
       this._handleFormSubmit(this._getInputValues());
-      this._popupForm.reset();
-      this._validator.toggleButtonState();
-      this.close();
     });
   }
+
+  submissionCompleate() {
```

Comment:
Function names should start with a **verb** in infinitive (there is such a point in the check list). It’s common in coding to do it so that functions could `speak` for themselves about what they do.